relay-server/src/actors/store.rs

```diff
@@ -558,6 +613,32 @@ struct AttachmentKafkaMessage {
     attachment: ChunkedAttachment,
 }
 
+/// Container payload for chunks of attachments.
+#[derive(Debug, Serialize)]
+struct ReplayRecordingChunkKafkaMessage {
+    /// Chunk payload of the replay recording.
+    payload: Bytes,
+    /// The replay id.
+    replay_id: EventId,
+    /// The project id for the current replay.
+    project_id: ProjectId,
+    /// The recording ID within the replay.
+    ///
+    /// The triple `(project_id, replay_id, id)` identifies a replay recording chunk uniquely.
```

Comment:
Is this actually true? The `id` alone is a unique identifier for the replay recording, but it's shared by all chunks. The tuple `(id, chunk_index)` is the unique identifier for a single chunk, plus it determines ordering.

Reply:
you are correct, fixing comment.